modules/core/src/main/scala/org/scalasteward/core/scalafix/MigrationAlg.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018-2019 Scala Steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scalasteward.core.scalafix
+
+import better.files.File
+import org.scalasteward.core.io.FileAlg
+import cats.implicits._
+import cats.Monad
+import io.circe.parser._
+import io.chrisdavenport.log4cats.Logger
+
+trait MigrationAlg[F[_]] {
+  def loadMigrations(migrationsFile: Option[File]): F[List[Migration]]
```

Comment:
@manuelcueto This function exposes to the rest of the program that there can be an optional file that contains a list of migrations. I think this is a detail which could be hidden inside the implementation of `MigrationAlg`. The rest of the program is more interested about available migrations for an `Update`, so a method like `def findMigrations(update: Update): F[List[Migration]]` would be more suitable here. WDYT?

Reply:
Makes sense.